.idea/OoS-Backend.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i believe settings files should be added to git ignore

Reply:
file was removed